doc/source/parameters/cfd/multiphysics.rst

```diff
@@ -13,6 +13,7 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
     set heat transfer                   = false
     set viscous dissipation             = false
     set buoyancy force                  = false
+    set DCDD shock capture              = false
```

Comment:
I think this parameter would be better in the stabilization section and not the multiphysics sectoin.

Reply:
Agreed